python/ray/_private/test_utils.py

```diff
@@ -64,6 +64,12 @@ def make_global_state_accessor(ray_context):
     return global_state_accessor
 
 
+def redis_mode():
```

Comment:
Can this also be more specific (as a test helper), e.g. `test_external_redis()`?